docker-compose.yml

```diff
@@ -14,6 +14,23 @@ services:
       - ./.env:/app/.env
     networks:
       - app-network
+  loadtest:
+    image: loadtest
+    build:
+      context: .
+      dockerfile: Dockerfile
+    user: app
+    env_file:
+      - .env
+    volumes:
+      - ./apps/:/app/apps
+      - ./packages/:/app/packages
+      - ./bin/:/app/bin
+      - ./package.json:/app/package.json
+      - ./.env:/app/.env
+    command: yarn workspace@app/condo cypress:loadtest
```

Comment:
it's better to create another docker-compose for load tests. It's not a part of default docker-compose scenarios